src/bash/report_campaigns/aggregate_campaign_tier

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+# Parse date argument
+date=20170228
+
+if [[ -n $1 ]]
+then
+    date=$1
+fi
+
+date_length=${#date}
+if [[ $date_length != 8 ]]
+then
+    echo 'Invalid date. Example: 20170228'
+    exit
+fi
+
+echo 'Aggregating for date: '$date
+
+# Results will be put here
+location=/cms/users/$USER/campaign_tier
```

Comment:
it is better to provide output area as external parameter to the script, otherwise it would be impossible to run this script multiple times since it will yield results in the same area. For instance, I may need to run this scripts to check the output and therefore want to put results into different output areas or I want to preserve my output area, etc.